homeassistant/components/light/mochad.py

```diff
@@ -37,8 +37,9 @@
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up X10 dimmers over a mochad controller."""
     devs = config.get(CONF_DEVICES)
+    controller = hass.data[mochad.DOMAIN]
     add_entities([MochadLight(
-        hass, mochad.CONTROLLER.ctrl, dev) for dev in devs])
+        hass, controller.ctrl_recv, dev) for dev in devs])
     return True
```

Comment:
Remove this. Nothing is checking this return value.

Reply:
Done